### PR TITLE
Fix #229

### LINKED ISSUE #229: Adding combatant after initiative is rolled reverses combatant order
ezgif com-optimize

- Add combatants to encounter
- Start encounter and set initiative
- Add new combatant (oops, forgot to add NPCs)
- Set initiative on new combatant
- List is reordered low to high (though initiative stays the same)

This occurs regardless of the type of combatant added (player, creature, quick-add), and regardless of initiative grouping (tested with None and By Name)

Tested on 1.5.5

### PR DESCRIPTION
**Pull request by cynicaloptimist**
Fixes #229

### CODE REVIEW

File: client/Encounter/Encounter.ts [hunk 1]
```diff
@@ -92,7 +92,7 @@ export class Encounter {
 
     private getCombatantSortIteratees(stable: boolean): ((c: Combatant) => number | string )[] {
         if (stable) {
-            return [c => c.Initiative()];
+            return [c => -c.Initiative()];
```
Comment (chrisbloom7) [inline client/Encounter/Encounter.ts]: 😆
